mne/time_frequency/tfr.py

```diff
@@ -2409,9 +2409,11 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     if copy is None:
         copy = baseline is not None
     data = rescale(data, times, baseline, mode, copy=copy)
+    coef = 10
```

Comment:
This all sounds good to me, but I think to save us some pain later we should do stuff like:
```suggestion
    # Assume that we have power unless shown otherwise
    coef = 10
```